BankManagerConsoleApp/Model/LinkedList.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  LinkedList.swift
+//  BankManagerConsoleApp
+//
+//  Created by 우롱차, 민성 on 2022/04/26.
+//
+
+import Foundation
+
+struct LinkedList<T> {
+    private var head: Node<T>?
+    private var tail: Node<T>?
+    var isNil: Bool {
+        if (head == nil) || (tail == nil) {
+            return true
+        } else {
+            return false
+        }
+    }
```

Comment:
조건문 자체가 Bool타입을 반환하므로 아래와 같이 작성하는것은 어떤가요? 또한 의미없는 괄호는 생략해주는 것을 권장드립니다.
```suggestion
    var isNil: Bool {
        return head == nil || tail == nil
    }
```

Reply:
오... 이런방법이... 감사합니다 개굴!